content/projects/data-science-specific/k-means-clustering/_index.md

```diff
@@ -28,14 +28,22 @@ title: K-Means Clustering Assignment
 
 ## Pre-requisites
 
-Read through the K-Means Tutorials at {{< contentlink path="topics/data-science-specific/clustering" >}} before starting this project.
+Read through the K-Means Tutorials at [content link](http://syllabus.africacode.net/topics/data-science-specific/clustering/) before starting this project.
 
-## Clustering whisky distilleries according to tasting Profiles
+## About the DataSet
 
-Data: [Whisky Tasting Profiles](whisky.csv)
+Goal: Clustering whisky distilleries according to tasting Profiles
```

Comment:
This looks like it is in an odd place. And the capitalisation is pretty strange.

Reply:
The heading says "About this dataset" but then below it there isn't information about the dataset but only about the goal of the project. Could we bring them into alignment?

Good also to check that some words aren't capitalised for no reason